core/whistle_apps-1.0.0/src/whapps_call_command.erl

```diff
@@ -945,49 +945,28 @@ b_page(Endpoints, Timeout, CIDName, CIDNumber, SIPHeaders, Call) ->
-    bridge_command(Endpoints, Timeout, wapi_dialplan:dial_method_single(), Call).
-bridge_command(Endpoints, Timeout, Strategy, Call) ->
-    bridge_command(Endpoints, Timeout, Strategy, <<"true">>, Call).
-bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Call) ->
-    bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, 'undefined', Call).
-bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, Call) ->
-    bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, 'undefined', Call).
-bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, SIPHeaders, Call) ->
-    bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, SIPHeaders, <<"false">>, Call).
-bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, SIPHeaders, IgnoreFoward, Call) ->
-    [{<<"Application-Name">>, <<"bridge">>}
-     ,{<<"Endpoints">>, Endpoints}
-     ,{<<"Timeout">>, Timeout}
-     ,{<<"Ignore-Early-Media">>, IgnoreEarlyMedia}
-     ,{<<"Ringback">>, wh_media_util:media_path(Ringback, Call)}
-     ,{<<"Dial-Endpoint-Method">>, Strategy}
-     ,{<<"Custom-SIP-Headers">>, SIPHeaders}
-     ,{<<"Ignore-Forward">>, IgnoreFoward}
-    ].
-
 bridge(Endpoints, Call) ->
-    Command = bridge_command(Endpoints, Call),
-    send_command(Command, Call).
+    bridge(Endpoints, ?DEFAULT_TIMEOUT_S, Call).
 bridge(Endpoints, Timeout, Call) ->
-    Command = bridge_command(Endpoints, Timeout, Call),
-    send_command(Command, Call).
+    bridge(Endpoints, Timeout, wapi_dialplan:dial_method_single(), Call).
 bridge(Endpoints, Timeout, Strategy, Call) ->
-    Command = bridge_command(Endpoints, Timeout, Strategy, Call),
-    send_command(Command, Call).
+    bridge(Endpoints, Timeout, Strategy, <<"true">>, Call).
 bridge(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Call) ->
-    Command = bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Call),
-    send_command(Command, Call).
+    bridge(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, 'undefined', Call).
 bridge(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, Call) ->
-    Command = bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, Call),
-    send_command(Command, Call).
+    bridge(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, 'undefined', Call).
 bridge(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, SIPHeaders, Call) ->
-    Command = bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, SIPHeaders, Call),
-    send_command(Command, Call).
+    bridge(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, SIPHeaders, <<"false">>, Call).
 bridge(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, SIPHeaders, IgnoreFoward, Call) ->
-    Command = bridge_command(Endpoints, Timeout, Strategy, IgnoreEarlyMedia, Ringback, SIPHeaders, IgnoreFoward, Call),
+    Command = [{<<"Application-Name">>, <<"bridge">>}
```

Comment:
I've been creating the `*_command` functions because sometimes we need the API payload itself to manipulate it more, publish it in a custom fashion, etc. I'd like to keep the builder functions available unless you have some convincing argument against it.